Sources/Orbit/Components/KeyValue.swift

```diff
@@ -15,76 +15,85 @@ import SwiftUI
 /// When the provided content is empty, the component results in `EmptyView` so that it does not take up any space in the layout.
 ///
 /// - Note: [Orbit.kiwi documentation](https://orbit.kiwi/components/keyvalue/)
-public struct KeyValue: View, PotentiallyEmptyView {
+public struct KeyValue<Key: View, Value: View>: View, PotentiallyEmptyView {
 
-    private let key: String
-    private let value: String
-    private let size: Size
+    @Environment(\.multilineTextAlignment) private var multilineTextAlignment
+    
+    @ViewBuilder private let key: Key
+    @ViewBuilder private let value: Value
 
     public var body: some View {
         if isEmpty == false {
-            KeyValueField(key, size: size) {
-                Text(value)
-                    .textSize(size.valueSize)
-                    .fontWeight(.medium)
+            VStack(alignment: .init(multilineTextAlignment), spacing: 0) {
+                key
+                    .textColor(.inkNormal)
+
+                value
+                    .textFontWeight(.medium)
+                    .textSize(.large)
                     .textIsCopyable()
             }
-            .accessibilityElement(children: .ignore)
-            .accessibility(label: .init(key))
-            .accessibility(value: .init(value))
+            .accessibility {
+                key
+            } value: {
+                value
+            }
             .accessibility(addTraits: .isStaticText)
+            .accessibility(.keyValue)
+            .textSize(.large)
         }
     }
     
     var isEmpty: Bool {
         key.isEmpty && value.isEmpty
     }
-}
-
-// MARK: - Inits
-extension KeyValue {
-
-    /// Creates Orbit ``KeyValue`` component.
+    
+    /// Creates Orbit ``KeyValue`` component with custom content.
     public init(
-        _ key: String = "",
-        value: String = "",
-        size: Size = .normal
+        @ViewBuilder value: () -> Value,
```

Comment:
Why is `value` first?

Reply:
This mimics native component custom viewbuilder inits, where (with exceptions) the order of parameters is switched (compared to variants where the non-viewbuilder parameters), so that the primary content or action associated with the component uses the first unnamed closure, while the rest uses closure labels.